deeppavlov/models/kbqa/wiki_parser.py

```diff
@@ -82,51 +96,67 @@ def execute_queries_list(self, parser_info_list: List[str], queries_list: List[A
         query_answer_types = []
         for parser_info, query in zip(parser_info_list, queries_list):
             if parser_info == "query_execute":
-                candidate_output = []
+                answers, found_rels, found_combs = [], [], []
                 try:
-                    what_return, query_seq, filter_info, order_info, answer_types, rel_types, return_if_found = query
+                    what_return, rels_from_query, query_seq, filter_info, order_info, answer_types, rel_types, \
+                    return_if_found = query
                     if answer_types:
                         query_answer_types = answer_types
-                    candidate_output = self.execute(what_return, query_seq, filter_info, order_info,
-                                                    query_answer_types, rel_types)
+                    answers, found_rels, found_combs = \
+                        self.execute(what_return, rels_from_query, query_seq, filter_info, order_info,
+                                     query_answer_types, rel_types)
                 except:
-                    log.warning("Wrong arguments are passed to wiki_parser")
-                wiki_parser_output.append(candidate_output)
+                    log.info("Wrong arguments are passed to wiki_parser")
```

Comment:
why?

Reply:
Wrong argument format can cause an error in query execution.